Pipfile

```diff
@@ -18,10 +18,16 @@ pywin32 = {version = "*",sys_platform = "== 'win32'"}
 [dev-packages]
 freezegun = "*"
 pytest = "*"
+
+flake8" = "*"
+isort = "*"
+yapf = "*"
+=======
```

Comment:
Looks like this is your issue, there shouldn't be any changes to the `Pipfile` here, and this is where the error is being thrown

Reply:
I tried to revert to the point when I could pass CI test, but it still didn't work.

Reply:
It looks like there's a separate issue going on with GitHub Actions, but this shouldn't revert the `master` branch's updates to the `Pipfile`

Reply:
Try and copy and paste the contents of the `Pipfile` and `Pipfile.lock` on the current `master` branch and run `pipenv sync --dev` to see if that updates things. I also re-ran the tests so the  output should be more informative